samples/betaFeatures.js

```diff
@@ -0,0 +1,608 @@
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+
+  //[END speechTranscribeMultilang]
+}
+
+function speechTranscribeMultilangGCS(gcsUri) {
+  // [START speechTranscribeMultilangGCS]
+
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+    audioChannelCount: 2,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .longRunningRecognize(request)
+    .then(data => {
+      const operation = data[0];
+      // Get a Promise representation of the final result of the job
+      return operation.promise();
+    })
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speechTranscribeMultilangGCS]
+}
+
+// WORD LEVEL CONFIDENCE
+function speechTranscribeWordLevelConfidence(speech_file) {
+  // [START speechTranscribeWordLevelConfidence]
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordTimeOffsets: true,
+    enableWordConfidence: true,
+  };
+
+  const audio = {
+    content: fs.readFileSync(speech_file).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        ` Transcription: ${transcription} \n Confidence: ${confidence}`
+      );
+
+      console.log(` Word-Level-Confidence:`);
+      const words = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      words.forEach(a => console.log(JSON.stringify(a, null, 4)));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speechTranscribeWordLevelConfidence]
+}
+
+function speechTranscribeWordLevelConfidenceGCS(gcsUri) {
```

Comment:
For python, I did not implement the GCS implementations of these 4 functions, since it was %98 similar to the file-implementation (This is just FYI).

Reply:
This might be a workaround for earlier issue with GCS..